### PR TITLE
feat: vue lab docs added

### PR DESCRIPTION
**Pull request by IamAfnanSk**
- added docs for vue labs as well
- docs links updated in `/docs/technologies` page

### CODE REVIEW

**Comment by IamAfnanSk** [comment]
@mehulmpt @rishabhrao @Pranavtechie PTAL

**Comment by IamAfnanSk** [comment]
@mehulmpt @rishabhrao @Pranavtechie ICYMI

File: docs/technologies/vue.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+# Install vitest and testing util
+cd /home/damner/code
+yarn add vitest@0.29.7 jsdom@21.1.1 @vue/test-utils@2.3.2 --dev
+mkdir -p /home/damner/code/__labtests
+ 
+# Move test file
+mv $TEST_FILE_NAME /home/damner/code/__labtests/all.test.js
+
+# vitest config file
+cat > /home/damner/code/__labtests/config.js << EOF
```
Comment (mehulmpt) [inline docs/technologies/vue.md]: Use `.labtests` folder (should be mentioned in https://teach.codedamn.com/docs/technologies/html-css as well. 

`__labtests` is deprecated now

**Review by mehulmpt — DISMISSED** [review]
One change, rest lgtm

**Pull request merged**